app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts
+  has_many :likes
+  has_many :comments
+
```

Comment:
- [ ] Kindy add a foreign key for your posts likes and comments eg ``has_many :posts, foreign_key: 'author_id'`` to complete the association.

Reply:
Dear, In the users model. How can I add a foreign key? it's a primary table that does not have any foreign key.

Reply:
@for-ashraf  What @danmainah means here is the foreign_key points to the posts table not the user's table. When you create this association and say `has_many: posts, foreign_key: 'user_id` it points to the `user_id` field that you add to the posts table. I have used `user_id` to explain since in your case you used `user_id` as the foreign key in your posts table which is also okay.

- [ ] So kindly add foreign keys just as requested by the first reviewer. If you need some more explanation let me know I will be willing to help

Reply:
Dear @peterrobert, I have made the following correction to the Users model: 
  has_many :posts, foreign_key: 'user_id'
  has_many :likes, foreign_key: 'user_id'
  has_many :comments, foreign_key: 'user_id'
Is that Ok?

Reply:
Yes this is perfect @for-ashraf